apps/omg_watcher/test/web/controllers/transaction_test.exs

```diff
@@ -17,11 +17,49 @@ defmodule OMG.Watcher.Web.Controller.TransactionTest do
   use ExUnit.Case, async: false
   use OMG.API.Fixtures
 
+  alias OMG.API.Crypto
   alias OMG.Watcher.TestHelper
 
-  describe "Controller.TransactionTest" do
+  @eth Crypto.zero_address()
+  @eth_hex String.duplicate("00", 20)
+
+  deffixture inputs do
+    [
+      %{
+        "amount" => 150,
+        "currency" => @eth_hex,
+        "blknum" => 2000,
+        "txindex" => 111,
+        "oindex" => 0,
+        "txbytes" => String.duplicate("00", 120)
+      },
+      %{
+        "amount" => 50,
+        "currency" => @eth_hex,
+        "blknum" => 5000,
+        "txindex" => 17,
+        "oindex" => 1,
+        "txbytes" => nil
+      }
+    ]
+  end
+
+  deffixture outputs(alice, bob) do
+    [
+      %{
+        "amount" => 97,
+        "owner" => alice.addr |> Crypto.encode_address!()
+      },
+      %{
+        "amount" => 100,
+        "owner" => bob.addr |> Crypto.encode_address!()
+      }
+    ]
+  end
+
+  describe "Controller.TransactionTest - transaction/:id" do
```

Comment:
I don't think we need the name of the test module in test description. It's displayed in console anyway.

Reply:
100%. I realized we're doing this all aroung Watcher. I'll create an chore to shave them all at once.